Dockerfile

```diff
@@ -1,189 +1,88 @@
-  && curl -O /usr/local/bin/install2.r https://github.com/eddelbuettel/littler/raw/master/inst/examples/install2.r \
-  && chmod +x /usr/local/bin/install2.r \
-  && rm -rf /var/lib/apt/lists/*
-
-## Download and install RStudio server & dependencies
-## Attempts to get detect latest version, otherwise falls back to version given in $VER
-## Symlink pandoc, pandoc-citeproc so they are available system-wide
-RUN apt-get update \
-  && apt-get install -y --no-install-recommends \
-    git \
-    libapparmor1 \
-    libcurl4-openssl-dev \
-    libedit2 \
-    libssl-dev \
-    lsb-release \
-    psmisc \
-    python-setuptools \
-    sudo \
-    rrdtool \
-    openssh-client \
-    libxml2-dev \
-    texinfo \
-    texlive \
-    texlive-latex-extra \
-    default-jre \
-    default-jdk \
-    bzip2 \
-    libbz2-dev \
-    libpcre3-dev \
-    liblzma-dev \
-    libglpk-dev \
-    libicu-dev \
-    libpng-dev \
-    libjpeg-dev \
-    libpoppler-cpp-dev \
-    libgeos-dev \
-    libgdal-dev \
-    libproj-dev \
-    libcairo2-dev \
-    lmodern \
-  && rm -rf /var/lib/apt/lists/* \
-  && wget -O libssl1.0.0.deb http://ftp.debian.org/debian/pool/main/o/openssl/libssl1.0.0_1.0.1t-1+deb8u6_amd64.deb \
-  && dpkg -i libssl1.0.0.deb \
-  && rm libssl1.0.0.deb
-
-# Install RStudio
-RUN wget -q https://download2.rstudio.org/rstudio-server-1.0.143-amd64.deb \
-  && dpkg -i rstudio-server-1.0.143-amd64.deb \
-  && rm rstudio-server-*-amd64.deb \
-
-  # Install pandoc
-  && ln -s /usr/lib/rstudio-server/bin/pandoc/pandoc /usr/local/bin \
-  && ln -s /usr/lib/rstudio-server/bin/pandoc/pandoc-citeproc /usr/local/bin \
-  && wget https://github.com/jgm/pandoc-templates/archive/${PANDOC_TEMPLATES_VERSION}.tar.gz \
-  && mkdir -p /opt/pandoc/templates && tar zxf ${PANDOC_TEMPLATES_VERSION}.tar.gz \
-  && cp -r pandoc-templates*/* /opt/pandoc/templates && rm -rf pandoc-templates* \
-  && mkdir /root/.pandoc && ln -s /opt/pandoc/templates /root/.pandoc/templates \
-
-  ## Configure R
-  && mkdir -p /etc/R \
-  && echo '\n\
+RUN mkdir -p /etc/R; echo '\n\
     \n .libPaths("~/R/library") \
     \n# Configure httr to perform out-of-band authentication if HTTR_LOCALHOST \
     \n# is not set since a redirect to localhost may not work depending upon \
     \n# where this Docker container is running. \
     \nif(is.na(Sys.getenv("HTTR_LOCALHOST", unset=NA))) { \
     \n  options(httr_oob_default = TRUE) \
     \n}' >> /etc/R/Rprofile.site \
-  && echo "PATH=\"${PATH}\"" >> /etc/R/Renviron \
-  && echo "r-libs-user=~/R/library" >> /etc/rstudio/rsession.conf \
-
-  ## Configure RStudio profile
-  && echo '\n\
-  \n[*] \
-  \nmax-memory-mb = 12288 \
-  \n' >> /etc/rstudio/profiles
+    && echo "PATH=\"${PATH}\"" >> /etc/R/Renviron \
+    && echo "r-libs-user=~/R/library" >> /etc/rstudio/rsession.conf \
+
+    ## Configure RStudio profile
+    && echo '\n\
+    \n[*] \
+    \nmax-memory-mb = 12288 \
```

Comment:
Interesting, I didn't realise we had this memory limit in the `Dockerfile`.

Reply:
I think it doesn't actually work: I think it's an R Studio pro feature that we implemented back when we tested pro